Packs/AzureKeyVault/Integrations/AzureKeyVault/AzureKeyVault.yml

```
@@ -0,0 +1,1171 @@
+      default: false
+      description: Permissions to secrets. If the Key Vault exists, you must supply
+        the previous secrets permissions inorder to keep them the same. Access policy
+        property. Default value is [get,list,set,delete,backup,restore,recover].
+      isArray: true
+      name: secrets
+      predefined:
+      - get
+      - list
+      - set
+      - delete
+      - recover
+      - backup
+      - restore
+      - purge
+      required: false
+      secret: false
+    - auto: PREDEFINED
+      default: false
+      description: Permissions to certificates. If the Key Vault exists, you must
+        supply the previous secret permissions inorder to keep them the same. Access
+        policy property. Default value is [get,list,update,create,import,delete,recover,backup,restore,managecontacts,manageissuers,getissuers,listissuers,setissuers,deleteissuers].
+      isArray: true
+      name: certificates
+      predefined:
+      - get
+      - list
+      - update
+      - create
+      - import
+      - delete
+      - recover
+      - backup
+      - restore
+      - managecontacts
+      - manageissuers
+      - getissuers
+      - listissuers
+      - setissuers
+      - deleteissuers
+      - purge.
+      required: false
+      secret: false
+    - auto: PREDEFINED
+      default: false
+      description: Specifies whether Azure Virtual Machines are permitted to retrieve
+        certificates stored as secrets from the key vault. If the Key Vault exists,
+        you must supply the previous value inorder to keep it the same. Default value
+        is True.
+      isArray: false
+      name: enabled_for_deployment
+      predefined:
+      - 'true'
+      - 'false'
+      required: false
+      secret: false
+    - auto: PREDEFINED
+      default: false
+      description: Specifies whether Azure Disk Encryption is permitted to retrieve
+        secrets from the vault and unwrap keys.If the Key Vault exists, you must supply
+        the previous value inorder to keep it the same. Default value is True.
+      isArray: false
+      name: enabled_for_disk_encryption
+      predefined:
+      - 'true'
+      - 'false'
+      required: false
+      secret: false
+    - auto: PREDEFINED
+      default: false
+      description: Specifies whether Azure Resource Manager is permitted to retrieve
+        secrets from the key vault. If the Key Vault exists, you must supply the previous
+        value inorder to keep it the same. Default value is True.
+      isArray: false
+      name: enabled_for_template_deployment
+      predefined:
+      - 'true'
+      - 'false'
+      required: false
+      secret: false
+    - auto: PREDEFINED
+      default: false
+      description: The default action when no rule from ip_rules and from vnet_subnet_id
+        match . For example,  If no ip_rules and vnet_subnet_id arguments are defined,
+        the access to the key vault from any ip address or virtual network will be
```

I will stop here, but please review in general as more comments like this one are likely
```suggestion
        the access to the key vault from any IP address or virtual network will be
```

---

Done (here and everywhere else).